internal/driver/interactive_test.go

```diff
@@ -132,7 +145,7 @@ var script = []string{
 	"f=-1;f=-2.5;check f=-2.5;f=0.0001;check f=0.0001",
 	"check ff=0;ff=-1.01;check ff=-1.01;ff=100;check ff=100",
 	"s=one;s=two;check s=two",
-	"ss=tree;check ss=tree;ss=;check ss;ss=forest;check ss=forest",
+	"ss=tree;check ss=tree;ss=forest;check ss=forest",
```

Comment:
I'm not 100% sure of this test right now, but I'm thinking the test as written originally checked for something useful (specifically, that passing the comman "ss=" didn't create problems), and am a bit unsure of the change here.

Reply:
The reason I deleted that was "sample_index=" was  a valid input before but not any more. You have to input "sample_index=0" for example. That's why "ss=" fails the test.

Reply:
I know including "ss=" will cause the test to fail as written, but I want to be sure that we don't eliminate a test that likely ensures that when an option does not get a value, interactive mode continues to allow commands after printing an error, rather than exiting immediately. 

Could an AllowRx instead be added to proftest.TestUI instead of removing "ss="?

Reply:
Yes, I actually added that case in the table driven tests.